content/blog-post/making-our-api-more-efficient.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "API Efficiency: How Lean Code Cut Energy Use by 99%"
```

Comment:
```suggestion
title: "Making our API more efficient for data consumers"
```
Slight tweak, as we can't easily estimate the energy savings. Better to talk about making things more efficient for users.